test/test.js

```diff
@@ -0,0 +1,43 @@
+var assert = require('chai').assert;
+var index = require('../index.js');
+var app = require('../app.js');
+var request = require('request');
+const https = require('https');
```

Comment:
I'd try to be consistent about using `var` or `const` for imports.  (And I'd probably prefer `const`, assuming that we're willing to use ES6 in this codebase.)